Cargo.toml

```diff
@@ -1,13 +1,12 @@
 [package]
 name = "header-append-filter"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["Daniel Grimm <dgrimm@redhat.com>"]
-edition = "2018"
+edition = "2021"
```

Comment:
was this intended? I wasn't aware that there is a 2021 edition yet?

Reply:
I HAVE NO IDEA :)

Isn't this the last modified year field?

Reply:
lol no this has special meaning :) it means Rust 2018, it's a language revision, similar to e.g. C++14

Reply:
lol

Reply:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-edition-field